modules/Library/library_manage_catalog_editProcess.php

```diff
@@ -52,25 +52,23 @@
             $row = $result->fetch();
             //Proceed!
             //Get general fields
-            $gibbonLibraryTypeID = $_POST['gibbonLibraryTypeID'];
-            $id = $_POST['id'];
-            $name = $_POST['name'];
-            $producer = $_POST['producer'];
-            $vendor = $_POST['vendor'];
-            $purchaseDate = null;
-            if ($_POST['purchaseDate'] != '') {
-                $purchaseDate = dateConvert($guid, $_POST['purchaseDate']);
-            }
-            $invoiceNumber = $_POST['invoiceNumber'];
-            $imageType = $_POST['imageType'];
+            $gibbonLibraryTypeID = $_POST['gibbonLibraryTypeID'] ?? '';
+            $id = $_POST['id'] ?? '';
+            $name = $_POST['name'] ?? '';
+            $producer = $_POST['producer'] ?? '';
+            $vendor = $_POST['vendor'] ?? '';
+            $purchaseDate = dateConvert($guid, $_POST['purchaseDate'] ?? null);
```

Comment:
`dateConvert` is deprecated since v16. It's probably best to rewrite `dateConvert` to `Gibbon\Services\Format::dateConvert` instead.

Reply:
Well spotted. I think we'll update these on a future pass, right now we're focusing on asingle change at a time 👍